Author: Robert James Clay <jame@rocasa.us>
Description: Correct minor spelling errors in LedgerSMB/DBObject/Payment.pm.
URL: https://github.com/ledgersmb/LedgerSMB/issues/1178
Origin: vendor
Last-Update: 2015-12-31
--- a/LedgerSMB/DBObject/Payment.pm
+++ b/LedgerSMB/DBObject/Payment.pm
@@ -281,7 +281,7 @@
 =item $payment->get_open_invoice()
 
 This function is an especific case of get_open_invoices(), because get_open_invoice() 
-can search for a specific invoice, wich can be searched by the $payment->{invnumber} 
+can search for a specific invoice, which can be searched by the $payment->{invnumber} 
 variable 
 
 =back
@@ -421,7 +421,7 @@
 This method gets a list of the open currencies inside the database, it requires that  
 $self->{account_class} (must be 1 or 2)  exist to work. 
 
-WARNING THIS IS NOT BEEING USED BY THE SINGLE PAYMENT SYSTEM.... 
+WARNING THIS IS NOT BEING USED BY THE SINGLE PAYMENT SYSTEM.... 
 
 =back
 
